src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts

```diff
@@ -1585,7 +1585,7 @@
     <message>
         <location filename="../pages/page_main.cpp" line="64"/>
         <source>Desktop Defaults</source>
-        <translation type="unfinished"></translation>
+        <translation>unfinished</translation>
```

Comment:
This needs to be finished, or reverted back to having the `type="unfinished"` tag.